src/mapwindow.jl

```diff
@@ -438,5 +438,7 @@ end
 default_shape(::Any) = identity
 default_shape(::typeof(median_fast!)) = vec
 
+@deprecate mapwindow(f, img, window, border, indices=axes(img)) mapwindow(f,img,window,border=border,indices=imginds)
```

Comment:
should this be `imginds=default_imginds(img)`? This doesn't look like it works. I expect this to throw an `UndefVarError`